R/ppmify.R

```diff
@@ -302,6 +305,36 @@ grid <- function (area, density) {
 
 }
 
+bias <- function (area, bias_points, offset.raster=FALSE){
+  
+  # calculate weights for integration points when using pseudo-observations as integration points
+  # weight = total study area / number of integration/quadrature points
+  
+  if (offset.raster){ # if offset raster is also provided, calculate sum of raster values
+    
+    a <- cellStats(area, sum, na.rm=TRUE)  
+    
+  } else {# calculate total study area
+    
+    a <- cellStats(area(area), sum, na.rm=TRUE)
+    
+  }
+  # calculate number of integration points
+  n <- nrow(bias_points)
```

Comment:
It would probably be sensible to convert to df before doing this (i.e. drag line 329 up here), so that it throws a meaningful error if the user passed something silly